Vision/README.md

```diff
@@ -1,6 +1,4 @@
-A New Focus for the W3C: Improving the Web’s Integrity
```

Comment:
I dropped this line entirely. The integrity point is made below, in "Mission", via "we must become more principled in our execution of the vision."

Reply:
Replicating comments from https://github.com/w3c/AB-public/pull/28#discussion_r1048813399:

[michaelchampion](https://github.com/michaelchampion) [5 hours ago](https://github.com/w3c/AB-public/pull/28#discussion_r1048860677)
Not sure I agree, but don't want to bikeshed about "integrity". My hope as an early author/editor of this document was to motivate W3C to "Pivot to Integrity". Probably not a good phrase, but the idea was to move W3C's vision from maximizing the POTENTIAL of the future web were were envisioning , and to the STEWARDSHIP the actual web we had built.

@[cwilso](https://github.com/cwilso) cwilso [4 hours ago](https://github.com/w3c/AB-public/pull/28#discussion_r1048895501)
I think that pivot is represented in this edit, but I'd welcome how to underscore it more.

@[michaelchampion](https://github.com/michaelchampion) michaelchampion [4 hours ago](https://github.com/w3c/AB-public/pull/28#discussion_r1048902289)
Ahh, I see you took out the "pillars" and replaced with "We will improve the fundamental integrity of the Web platform. The Web will not only grow in scope and importance in our lives; it will grow in respect for its users, grow in the trust of its users, grow in its inclusion of all humanity as its users."

That's fine

@[dwsinger](https://github.com/dwsinger) dwsinger [1 hour ago](https://github.com/w3c/AB-public/pull/28#discussion_r1049046029)
I think it was supposed to be a one-line title or subtitle; I'd rather have a title that says something about what follows. Let's look for a snappy/catchy informative title that keeps the idea of values being pivotal? (Not a strong objection)